next/components/Atoms/SelectField.tsx

```diff
@@ -0,0 +1,160 @@
+import ChevronDown from '@assets/images/chevron-down.svg'
+import FieldWrapper from '@components/Molecules/FieldWrapper'
+import { Listbox } from '@headlessui/react'
```

Comment:
I see headlessui being used - not a problem with that, just wondering what is our consensus on these libs. Is it "react-aria unless it does not have the right stuff, then pick from others, then code your own" ?

Reply:
This component was just copied from Marianum. We plan to refactor it to use react-aria, but it was not task for this PR. I know it is not ideal to introduce "more work" into the project, but as the refactoring will be the same for both project, I think it is reasonable quick solution to use this component.